performant-software/arc-visualization
# made bubble colors more accessible

### What does this PR do?
- Changes discipline bubble colors (which previously were red and did not meet web content accessibility guidelines) to a color palette that was more accessible. 

### What issues does it address?
- Closes #4 

### How to test:
- Access staging server (see slack for details)
- Go to arc-visualization directory 
`[bigdiva@bigdiva-staging ~]$ cd arc-visualization/`
- Run `$ rvmsudo unicorn_rails -o 192.237.180.236 -p 80`
- Access the BigDiva page by going to 192.237.180.236
- Notice the circles next to "Visualize By:" under the BigDiva logo.
- Note that there is no red circle, rather a vivid amber to the right of "Discipline".
- Click on "Discipline" and note that the bubbles that appear are not red, but the same vivid amber.
- To compare what the red used to look like, go to http://bigdiva.org/
